olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/ConsumerProperties.h

```diff
@@ -23,33 +23,67 @@
 #include <string>
 #include <vector>
 
-#include <boost/optional.hpp>
 #include <olp/dataservice/read/DataServiceReadApi.h>
+#include <boost/optional.hpp>
```

Comment:
According to clang Google style "Your project's .h files" have to be **after** "Other libraries' .h files". So include files order was ok and you should not change the order. Please refer to https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes for more details.

Reply:
Thank you!
Fixed